sdks/python/apache_beam/io/fileio.py

```diff
@@ -1081,12 +1084,20 @@ def __init__(self,
         mime_type='text/plain',
         compression_type=compression_type)
     self.append_trailing_newlines = append_trailing_newlines
-
+    self.header = header
```

Comment:
Note that append_trailing_newlines didn't have _ prefix. Anyway, updated both now.